### DISCUSSION TITLE
Implement Mythic Points in Non-Latest Raids

### DISCUSSION THREAD

**Discussion by NinesCC** [chromiecraft/chromiecraft#6894] [Suggestions]
Enable **Mythic Mode** in SSC with release values (pre-nerf / pre-March 20, 2024) and let people buy gear the same way as Magtheridon and Gruul. Have the new nerfed version be the **Normal Mode**.

SSC is too easy with the huge nerfs, while still being current tier. Throw guilds a bone so there is something to look forward at least.

**Comment by Mcadeez** [comment]
Yes please.

**Comment by Never6** [comment]
This would be a great, in addition I think it would be nice to have reward system similar to how Gruul and Mag were because some people have done SSC for weeks and still need items (belt next week copium)

**Comment by AcetylGer** [comment]
Any updates on this?

**Comment by Spoon722** [comment]
It would be nice to have a way to do the fights with the original tuning, if everyone has gear from the latest tier you can essentially ignore all mechanics on content that has been nerfed.

**Comment by vvranceanu** [comment]
Devs

For the love of god, please don't transform this into a fun-server. It's bad enough that 80% of the server had DST. People are currently raiding SSC cause they need the trinkets, if you implement mythic points it will be empty in 3 weeks and the few people that still need it can't join groups cause there are no groups. You will ruin the raiding "economy".

I joined this server expecting Wotlk. This is slowly transforming into a big participation trophy. Let people that have been playing for 10 years on end and are bored of it to play it on PTR with GM commands, i wanna enjoy the game. I wanna enjoy that Tsunami after i get it after 5 months of PUGs. Not everyone needs to have bis, that is the nature of the game. Its part of the challenge to build something without having bis every step of the way...

Take a step back and realize you went from attunements and challenging heroics to now what is to be a in-and-out bis session. This is not throwing anyone a bone but the raid-loggers. Every single new person we've had in guild was confused about the Mythic points on Gruul. Its a kitsch, its cheap, its unexpected from anyone joining a Wotlk server.

Please stop catering to the easy-reward group, they will keep making demands over demands, as shown. The game is not fun for them anymore, they wanna play another game. Let people that wanna play Wotlk and joined for Wotlk play Wotlk. Please...

**Reply by AcetylGer** [reply]
Nobody is asking for "easy-reward solutions". The mythic mode giving mythic points would be tuned akin to mythic Mag/Gruul.

You Argument about nobody running the content is invalid since there are still 2 groups running mythic Gruul/Mag every week for fun.
You are still able to run SSC for 5 Months with a pug for your item because most groups wont attempt Mythic and will just clear the normal modes. And if your guild is able to clear mythic TK/SSC every week then you will most likely be raiding Hyjal and soon BT and will just skip the items you havent gotten yet.

**Reply by vvranceanu** [reply]
ty for providing your own perspective on how the mythic points have no place. there is no scenario in which you need to buy gear with currency: you either raid the latest content and you skip older loot anyway or you cant do mythic, so have no access to mythic points.

appreciate the input!

**Reply by AcetylGer** [reply]
I dont think you "need" the points in older content but it makes farming transmogs easier, that would be one scenario where the points are useful, the other would be the weird way too powerful items that are dropping in T4 and are bis till sunwell.
There is more of a "need" when talking about current tier content mythic points.

Overall my feedback is that mythic difficulty make the raids interesting for a longer time. I dont think anyone would run mythic gruul/mag if there were no mythic points tho. It enables better gearing paths for alts and the increased difficulty keeps the fights interesting even with a group of mains.

**Reply by vvranceanu** [reply]
So you're saying that mythic difficulty is nothing but a system to get the bis for the already successful players that can clear mythic content and latest releases frustrated that their bis didn't drop/didn't won the roll yet and it would not actually present interest cause of the difficulty as this already assumed duplicitous post tries to hide behind, has nothing to do with a healthy server(as stated, it will only reduce drastically the pool of players raiding older content)

I'm glad we're on the same page!

Indeed, that is the design of the game, possibly the cunning developers were thinking the same thing when introducing powerful quirky items in lower tier content together with the attunements, such that their servers dont become baren wastelands of raidloggers in parallel with people that cant find a raid spot cause no one logs in besides to do the latest raid!

Appreciate your points, again!

**Reply by Anwyyn** [reply]
1st. I don't understand, why is it "bad enough" that "80%" (not sure if you can confirm this info) had DST?

2nd "if you implement mythic points it will be empty in 3 weeks" Think that after 1 year later that Gruul/Mag were realeased and you  still can find several guilds and pugs running both normal a mythic versions has already proven this point to not be valid.

3rd  "I wanna enjoy that Tsunami after i get it after 5 months of PUGs" That is the best part of mythichs, that they are OPTIONAL. If you want to farm 1 item for 5 months, go for it and keep lookin for normal pug runs. You won't be forced to do mythics, and again, normal Gruul runs being alive after 1 year has proven this point to be true.

4rd "Not everyone needs to have bis" Current tunning meters on release require your raid to be on BiSes, and even those bisses might not be enough. I.E. Archi 1 shotting main tanks on full BiS gear.

5th "Its part of the challenge to build something without having bis every step of the way" A challenge that might not suit for everyone, how challeging does soemthing like tunning might be is just personal opinion. Yes, you can't please everyone, but still try to be better for the majority. Current discord discussion has at least shown that there is a big group of people who are totally against of having raids way harder from what it was TBCC release.

6th "its unexpected from anyone joining a Wotlk server" This is a Pserver, with many "added" things from what retail/classic had, just like basically any other pservers. Plus, it was already stated that CC is not willing to be just a copy of retail.

7th "they will keep making demands over demands" as far as those "demands" are fair and suggested for the good of the community, I am personally ok with it.

8th "The game is not fun for them anymore, they wanna play another game" I am not sure about where your info came from, but at least I know some people who might totally disagree with this point, or how it is stated.

**Reply by heyitsbench** [reply]
> I'm glad we're on the same page!

No one wants to engage with someone that can't discuss a topic in good faith.

**Reply by vvranceanu** [reply]
> > I'm glad we're on the same page!
> 
> No one wants to engage with someone that can't discuss a topic in good faith.

It's what i was hoping when i prefaced everything with "Devs" :). Still they wanna parrot the same basic ideas they do on discord even here. What can you do... we live in a society. Can you comprehend people that can't grasp the complexities of pressing precisely 2 buttons get to dictate what the balance and features should be? It's like letting toddlers decide that candy is for dinner ^_^. It's insane, i know

**Reply by heyitsbench** [reply]
[Learn to play nice.](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines)

**Reply by Never6** [reply]
> Devs
> 
> For the love of god, please don't transform this into a fun-server. It's bad enough that 80% of the server had DST. People are currently raiding SSC cause they need the trinkets, if you implement mythic points it will be empty in 3 weeks and the few people that still need it can't join groups cause there are no groups. You will ruin the raiding "economy".
> 
> I joined this server expecting Wotlk. This is slowly transforming into a big participation trophy. Let people that have been playing for 10 years on end and are bored of it to play it on PTR with GM commands, i wanna enjoy the game. I wanna enjoy that Tsunami after i get it after 5 months of PUGs. Not everyone needs to have bis, that is the nature of the game. Its part of the challenge to build something without having bis every step of the way...
> 
> Take a step back and realize you went from attunements and challenging heroics to now what is to be a in-and-out bis session. This is not throwing anyone a bone but the raid-loggers. Every single new person we've had in guild was confused about the Mythic points on Gruul. Its a kitsch, its cheap, its unexpected from anyone joining a Wotlk server.
> 
> Please stop catering to the easy-reward group, they will keep making demands over demands, as shown. The game is not fun for them anymore, they wanna play another game. Let people that wanna play Wotlk and joined for Wotlk play Wotlk. Please...

That's nice for you
But I farmed ssc for 5months too and can count on one hand how many tsunami's I've seen (with fingers to spare 😀 )
Saying the raid "would be empty" while that might be true it has no correlation with people being able to remove rng on items
"Easy rewards" you still have to complete the raid, hell even multiple times. And you have to do that with tuning 
"Let the people who want to play wotlk, play Wotlk"
Yeah, people willingly join a 3.3.5 client progressing through TBC raids which doesn't even have northrend continent available, to play wrath of the lich king
Not whitemane or warmane
200pop CC
So many delusional takes its amazing you can even use a computer

**Reply by vvranceanu** [reply]
> > Devs
> > For the love of god, please don't transform this into a fun-server. It's bad enough that 80% of the server had DST. People are currently raiding SSC cause they need the trinkets, if you implement mythic points it will be empty in 3 weeks and the few people that still need it can't join groups cause there are no groups. You will ruin the raiding "economy".
> > I joined this server expecting Wotlk. This is slowly transforming into a big participation trophy. Let people that have been playing for 10 years on end and are bored of it to play it on PTR with GM commands, i wanna enjoy the game. I wanna enjoy that Tsunami after i get it after 5 months of PUGs. Not everyone needs to have bis, that is the nature of the game. Its part of the challenge to build something without having bis every step of the way...
> > Take a step back and realize you went from attunements and challenging heroics to now what is to be a in-and-out bis session. This is not throwing anyone a bone but the raid-loggers. Every single new person we've had in guild was confused about the Mythic points on Gruul. Its a kitsch, its cheap, its unexpected from anyone joining a Wotlk server.
> > Please stop catering to the easy-reward group, they will keep making demands over demands, as shown. The game is not fun for them anymore, they wanna play another game. Let people that wanna play Wotlk and joined for Wotlk play Wotlk. Please...
> 
> That's nice for you But I farmed ssc for 5months too and can count on one hand how many tsunami's I've seen (with fingers to spare 😀 ) Saying the raid "would be empty" while that might be true it has no correlation with people being able to remove rng on items "Easy rewards" you still have to complete the raid, hell even multiple times. And you have to do that with tuning "Let the people who want to play wotlk, play Wotlk" Yeah, people willingly join a 3.3.5 client progressing through TBC raids which doesn't even have northrend continent available, to play wrath of the lich king Not whitemane or warmane 200pop CC So many delusional takes its amazing you can even use a computer

i like your avatar
